app/controllers/ContactPhoneController.scala

```diff
@@ -55,36 +53,26 @@ class ContactPhoneController @Inject() (
 
   private val form = formProvider()
 
-  private def data(mode: Mode, regime: Regime, form: Form[String])(implicit request: DataRequest[AnyContent]): Option[JsObject] = {
+  private def data(mode: Mode, regime: Regime, form: Form[String])(implicit request: DataRequest[AnyContent]): JsObject = {
 
     val suffix       = isBusinessOrIndividual()
     val orIndividual = if (suffix == "individual") Some("") else None
-    request.userAnswers.get(ContactNamePage).orElse(orIndividual).map {
-      name =>
-        Json.obj(
-          "form"      -> request.userAnswers.get(ContactPhonePage).fold(form)(form.fill),
-          "regime"    -> regime.toUpperCase,
-          "name"      -> name,
-          "pageTitle" -> s"contactPhone.title.$suffix",
-          "heading"   -> s"contactPhone.heading.$suffix",
-          "hintText"  -> hintWithNoBreakSpaces(),
-          "action"    -> routes.ContactPhoneController.onSubmit(mode, regime).url
-        )
-    }
+    val name         = request.userAnswers.get(ContactNamePage).orElse(orIndividual)
```

Comment:
@BriWak It's an existing code, We should move away from orElse and place the correct condition
Can we fix this